charts/cinder-csi-plugin/values.yaml

```diff
@@ -30,7 +30,7 @@ csi:
   plugin:
     image:
       repository: docker.io/k8scloudprovider/cinder-csi-plugin
-      tag: v1.18.0
+      tag: latest
```

Comment:
latest is fine for master, but I am thinking that should we should remember update this to release-1.19 branch